backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/ddl/public/functions/docs_sales_invoice_description.sql

```diff
@@ -0,0 +1,106 @@
+DROP FUNCTION IF EXISTS de_metas_endcustomer_fresh_reports.docs_sales_invoice_description(IN record_id   numeric,
```

Comment:
Same, regarding the place.
Shall be in here https://github.com/metasfresh/metasfresh/tree/9b07692c02cf5041238b7a5300edf47433ffdb9c/backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions.